pkg/user/router.go

```diff
@@ -7,6 +7,7 @@ import (
 
 func Routes(r *gin.Engine, authenticationMiddleware middleware.AuthenticationMiddleware, authorizationMiddleware middleware.AuthorizationMiddleware, handler Handler) {
 	r.POST("/users", handler.SignUp)
+	r.DELETE("/users", handler.SignOut)
```

Comment:
Let's say a user has checked the "Remember me" box when logging in but both the access token and the refresh token is expired. So when sending the logout request we'll simply reset the cookies rather than forcing the user to login again to do so.